src/desktop/models/user.js

```diff
@@ -5,107 +5,105 @@
  * DS207: Consider shorter variations of null checks
  * Full docs: https://github.com/decaffeinate/decaffeinate/blob/master/docs/suggestions.md
  */
-let _User;
-const _ = require('underscore');
-const Backbone = require('backbone');
-const { SESSION_ID } = require('sharify').data;
-const { Geo } = require('./mixins/geo');
-const { UserRelations } = require('./mixins/relations/user');
+let _User
+const _ = require("underscore")
+const Backbone = require("backbone")
+const { SESSION_ID } = require("sharify").data
+const { Geo } = require("./mixins/geo")
+const { UserRelations } = require("./mixins/relations/user")
 
 // Base User model for shared functionality between
 // CurrentUser and LoggedOutUser
-export default (_User = (function() {
+export default _User = (function () {
   _User = class User extends Backbone.Model {
     static initClass() {
-      _.extend(this.prototype, Geo);
-      _.extend(this.prototype, UserRelations);
+      _.extend(this.prototype, Geo)
+      _.extend(this.prototype, UserRelations)
 
       this.prototype.priceBuckets = [
-        { display: 'Under $500', value: '-1:500' },
-        { display: 'Under $2,500', value: '-1:2500' },
-        { display: 'Under $5,000', value: '-1:5000' },
-        { display: 'Under $10,000', value: '-1:10000' },
-        { display: 'Under $25,000', value: '-1:25000' },
-        { display: 'Under $50,000', value: '-1:50000' },
-        { display: 'Under $100,000', value: '-1:100000' },
-        { display: 'No budget in mind', value: '-1:1000000000000' }
-      ];
-    }
-
-    // This refreshes the user data in the session so that saved data
-    // will stay in sync on reloads
-    refresh(options) {
```

Comment:
The only change in this file is deleting this `refresh` method. The rest of the diff is just Prettier updates.